nerfstudio/configs/method_configs.py

```diff
@@ -124,7 +117,7 @@
     pipeline=VanillaPipelineConfig(
         datamanager=VanillaDataManagerConfig(
             dataparser=NerfstudioDataParserConfig(),
-            train_num_rays_per_batch=4096,
+            train_num_rays_per_batch=8000,
```

Comment:
Is 8192 too big? Powers of 2 are nice

(if some `nerfacto-big` tuning happened in this PR we could also tweak the title perhaps)

Reply:
sorry my brain works in base 10